### PR TITLE
Merge releases/m71 to master

### PR DESCRIPTION
**Pull request by jamescadd**
Documentation is now complete enough for a first pass and ready to be merged to master for publication.

### CODE REVIEW

File: how-to-audio-processing.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+```C#
+private void DisplayAudioBufferEvent(
+            ref int totalSamples,
+            String type,
+            Org.WebRtc.IAudioBufferEvent evt)
+{
+    var buffer = evt.Buffer;
+    var data = buffer.Channel(0);
+    var array = data.Data.ToArray();
```
Comment (robin-raymond) [inline how-to-audio-processing.md]: var array = new Int16[data.Length];
            data.GetData(array);

File: how-to-audio-processing.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+    double scale = Math.Pow(2, 10);
+
+    for (int index = 0; index < array.Count(); ++index, ++totalSamples)
+    {
+        array[index] = (short)(Math.Sin(totalSamples) * scale);
+    }
+
+    data.Data = array;
```
Comment (robin-raymond) [inline how-to-audio-processing.md]: data.SetData(dataArray);

**Review by robin-raymond — CHANGES_REQUESTED** [review]

File: how-to-audio-processing.md (hunk 1 above)
Comment (jamescadd) [reply how-to-audio-processing.md]: Good catch, corrected.

File: how-to-audio-processing.md (hunk 2 above)
Comment (jamescadd) [reply how-to-audio-processing.md]: Same

**Review by robin-raymond — APPROVED** [review]

File: azure-devops-build.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+Script path:
+$(Build.SourcesDirectory)\scripts\run.py
+Argument to  
+NugetUwpCI --buildOutputPath $(Build.BinariesDirectory)\Build --nugetOutputPath $(Build.BinariesDirectory)\Nuget --gnOutputPath $(Build.BinariesDirectory)\GN --manualNugetVersionNumber "1.71.0.$(Build.BuildId)"
+Working directory to $(Build.SourcesDirectory)
+Custom condition: 
+eq(variables['config.packNuget'], 'true')
+12. Create a WebRtc Feed
+Follow the bellow link to create a feed in the Artifact and connect it to the nuget package
+https://docs.microsoft.com/en-us/azure/devops/artifacts/get-started-nuget?view=azure-devops&tabs=new-nav#create-a-feed
```
Comment (Joshua9115) [inline azure-devops-build.md:91]: good

File: how-to-audio-device-selection.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
+# Audio device enumeration and selection
```
Comment (Joshua9115) [inline how-to-audio-device-selection.md:1]: my name

File: how-to-audio-device-selection.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
+# Audio device enumeration and selection
+
+While the WebRTC APIs in Chromium originally supported enumeration and selection of the audio capture (microphone) and render (speaker) devices, this API was removed in later releases.  This behavior is now handled in system-specific call for all native platforms, but is not part of the core WebRTC engine.  WebRTC on Windows will use the system default capture and render devices if they are not specified.
+
+The following code from the `MainViewModel.cs` file of the `PeerCC` sample demonstrates enumeration & setting of capture and render devices:
```
Comment (Joshua9115) [inline how-to-audio-device-selection.md:5]: atlerviper

Comment (Joshua9115) [inline how-to-audio-device-selection.md:5]: ATLERVIPER

**Review by Joshua9115 — COMMENTED** [review]
Thank you

**Pull request merged**